Makefile

```diff
@@ -99,6 +99,15 @@ MCU_ISP_PIN         ?= 13
 LED_CONN_PIN        ?= 0
 # GPIO pin used for "serial activity" LED, active low
 LED_SERIAL_PIN      ?= 14
+#
+# Default settings for access over TCP/IP connections
+#
+# Modes are 0 (unsecure), 1 (disabled), 2 (secure)
+#
+PORT1_MODE		?=	0
+PORT1_PORTNUMBER	?=	23
+PORT2_MODE		?=	0
+PORT2_PORTNUMBER	?=	2323
```

Comment:
There is a PR #205 that going to get merged to change the ports via the web UI. I'd rather not have additional settings in the Makefile too. It gets very confusing when there are settings in the Makefile 'cause it's not obvious at all which take effect when. There are settings for Wifi in the Makefile and that has been a confusing mess, and those are necessary because without wifi it's hard to connect after flashing, so at least there's a good reason for them.